test/multiverse/suites/redis/redis_instrumentation_test.rb

```diff
@@ -59,6 +59,8 @@ def test_records_metrics_for_connect
     end
 
     def test_records_connect_tt_node_within_call_that_triggered_it
+      skip_for_redis_clustering_gem
```

Comment:
In the end, it seemed the least intrusive to add skips for the six tests that are not compatible with redis-clustering instrumentation:
* We can't test with `Redis::Cluster.new` because we don't have a cluster mode enabled service running for our tests, so we're faking the "cluster" testing regardless
* We don't have a different configuration option for `redis-clustering` than we do for `redis`, so the prepend/chain configurations seemed excessive
* Skipping the tests feels easier to read and understand the feature disparity between the two gems quickly

Open to further discussion to take a different approach!